.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,228 @@
+name: "pull-request"
+
+on:
+  push:
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+
+env:
+  APP_CONFIG_ECM_HOST: ${{ secrets.PIPELINE_ENV_URL }}
+  ADMIN_EMAIL: ${{ secrets.PIPELINE_ADMIN_USERNAME }}
+  ADMIN_PASSWORD: ${{ secrets.PIPELINE_ADMIN_PASSWORD }}
+  AWS_REGION: "eu-west-2"
+  CONTENT_CE_DIST_PATH: "./dist/content-ce"
+  APP_CONFIG_PROVIDER: ECM
+  APP_CONFIG_AUTH_TYPE: BASIC
+  APP_CONFIG_OAUTH2_HOST: http://localhost:4200/auth/realms/alfresco
+  PLAYWRIGHT_E2E_HOST: "http://localhost:4200"
+  APP_CONFIG_OAUTH2_CLIENTID: alfresco
+  APP_CONFIG_PLUGIN_AOS: true
+  APP_CONFIG_PLUGIN_CONTENT_SERVICE: true
+  APP_CONFIG_PLUGIN_FOLDER_RULES: true
+  APP_CONFIG_ENABLE_MOBILE_APP_SWITCH: true
```

Comment:
can we turn this off so it does not nag us ?

Reply:
it might mess the e2e I presume

Reply:
Oh right good idea, I will change it